### PR TITLE
test: rectify failing tests

### PR DESCRIPTION
**Pull request by blunteshwar**
## Description

Some tests were failing on main branch on local. This PR will aim at rectifying those failing tests.

## Types of changes

-   [x] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [ ] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

-   [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [x] My code follows the code style of this project.
-   [x] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [x] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [x] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [x] I have reviewed at the Accessibility Practices for this feature, see: [Aria Practices](https://www.w3.org/TR/wai-aria-practices/)

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.

### CODE REVIEW

File: tools/grid/test/grid.test.ts [hunk 1]
```diff
@@ -51,7 +51,8 @@ describe('Grid', () => {
 
         expect(el.tabIndex).to.equal(0);
 
-        el.focus();
+        await sendKeys({ press: 'Tab' });
+        await sendKeys({ press: 'Tab' });
```
Comment (rubencarvalho) [inline tools/grid/test/grid.test.ts:56]: Can we have an assertion or comment here to understand what these Tabs are doing?

File: tools/grid/test/grid.test.ts [hunk 2]
```diff
@@ -77,7 +78,8 @@ describe('Grid', () => {
...
```
Comment (rubencarvalho) [inline tools/grid/test/grid.test.ts:83]: Same here 😄

**Review by rubencarvalho — CHANGES_REQUESTED** [review]
Thank you for fixing these :)

File: tools/grid/test/grid.test.ts (hunk 1 above)
Comment (blunteshwar) [reply tools/grid/test/grid.test.ts:56]: Instead of calling focus() function, we are focusing using Tab key

Comment (rubencarvalho) [reply tools/grid/test/grid.test.ts:56]: can we add an assertion to check if the element is indeed focused?

Comment (blunteshwar) [reply tools/grid/test/grid.test.ts:56]: The code after line 56 is checking for exactly the same!

**Review by rubencarvalho — APPROVED** [review]

**Pull request merged**